check_exchange_rates.py

```diff
@@ -15,40 +19,48 @@
 # Receiver
 recipients = {
     "elokuuu0321@gmail.com": [
-        {"currency": "澳大利亚元 AUD", "threshold": 4.7},
-        {"currency": "美元 USD", "threshold": 7.2},
+        {"currency": "AUD", "threshold": 4.75},
+        {"currency": "USD", "threshold": 7.2},
     ],
     "harryqin1999@gmail.com": [
-        {"currency": "澳大利亚元 AUD", "threshold": 4.7},
-        {"currency": "日元 JPY", "threshold": 0.05},
-        {"currency": "美元 USD", "threshold": 7.2},
+        {"currency": "AUD", "threshold": 4.7},
+        {"currency": "JPY", "threshold": 0.05},
+        {"currency": "USD", "threshold": 7.2},
     ],
 }
 
-# 
+# Check exchange rates
 def check_exchange_rates():
     try:
-        response = requests.get(API_URL)
+        # Attempt to fetch data with a 10-second timeout
+        response = requests.get(API_URL, timeout=10)
+        response.raise_for_status()
         data = response.json()
+        print("\n")
+
 
         for email, currencies in recipients.items():
             for settings in currencies:
                 currency_name = settings["currency"]
                 threshold = settings["threshold"]
 
-                # Check rate
                 for currency in data['body']:
```

Comment:
Could you please create a currency class and convert the body to a list of currency class, rather than accessing the data via json body? Thanks